macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -436,34 +452,69 @@ fileprivate extension Unicode.Scalar {
 
 fileprivate extension NSColor {
 	/// the text color of the text in the initials view
-	///
-	/// - note: this value doesn't change for dark mode by design as our default background color table
-	/// only provides colors designed to be used with white text, even on dark mode
-    static let initialsViewTextColor: NSColor = .white
 
 	/// the table of background colors for the initials views
-	static let avatarBackgroundColors: [NSColor] = [
-		Colors.Palette.cyanBlue10.color,
-		Colors.Palette.red10.color,
-		Colors.Palette.magenta20.color,
```

Comment:
if there are palette colors that aren't being used anymore.... should we delete them from the asset catalog?

Reply:
I like the idea, it would save space as well, @rdeassis  any objections?

Reply:
No objections at all if they're not being used!

Reply:
It seems like these are still being references in iOS